src/witan/send/adroddiad/vega_specs/lines.clj

```diff
@@ -15,57 +15,48 @@
    using format `fmt` for each summary (default \"%,.0f\")
    after applying (optional) function `f` to it.
    (So for percentages specify `:fmt \"%.0f%%\" :f (partial * 100)`.)"
-  [[orl irl y iru oru] & {:keys [fmt f]
-                          :or   {fmt "%,.0f"
-                                 f   identity}}]
-  (apply format
-         (apply format "%s (%s (%s↔%s) %s)" (repeat 5 fmt))
-         (map f [y orl irl iru oru])))
+  ([xs] (five-number-summary-string {} xs))
+  ([{:keys [fmt f] :or {fmt "%,.0f", f identity}} [orl irl y iru oru]]
+   (apply format
+          (apply format "%s (%s (%s↔%s) %s)" (repeat 5 fmt))
+          (map f [y orl irl iru oru]))))
 
 (comment
-  (five-number-summary-string ["p05" "q1" "median" "q3" "p95"] :fmt "%s")
+  (five-number-summary-string {:fmt "%s"} ["p05" "q1" "median" "q3" "p95"])
   ;; => "median (p05 (q1↔q3) p95)"
-  (five-number-summary-string [0.3 0.1 0.2 0.4 0.5] :fmt "%.0f%%" :f (partial * 100))
+  (five-number-summary-string [3.0 1.0 2.0 4.0 5.0])
+  ;; => "2 (3 (1↔4) 5)"
+  (five-number-summary-string {:fmt "%.0f%%", :f (partial * 100)} [0.3 0.1 0.2 0.4 0.5])
```

Comment:
I miss the cuteness of the trailing keywords, but I see why you did it like this. `partial` isn't quite as loved in clojure as `#()` is